libtcmu_config.c

```diff
@@ -367,19 +368,23 @@ int tcmu_load_config(struct tcmu_config *cfg, const char *path)
 	fd = open(path, O_RDONLY);
 	if (fd < 0) {
 		tcmu_err("Failed to open file '%s'\n", path);
-		return -1;
+		res = -1;
```

Comment:
This is just a matter of coding style, but If I write this, I would have initialized 'res=-1;' and at the end, before label 'out:' I would have assigned it to 'res=0;'

But as of today it looks okay, in the future if someone introduces more check in this function has to duplicate "res=-1" in every { }